includes/Dashboard/Templates/Settings.php

```diff
@@ -17,9 +17,9 @@ class Settings {
     /**
      * Loading autometically when class initiate
      *
+     * @return void
```

Comment:
@zamandevs why did you move this line for every method?

Reply:
didn't notice that. I maintain phpstorm  plugin for Wordpress. when reformatted whole code it occurred.